src/main/webapp/scripts/script-session.js

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * function openSessionInfo() displays the div container
+ * that has information about the session.
+ */
+function openSessionInfo() {
+  document.getElementById('session-info-div').style.display = 'block'; 
+}
+
+/**
+ * function closeSessionInfo() closes the div container
+ * that has information about the session.
+ */
+function closeSessionInfo() {
+  document.getElementById('session-info-div').style.display = 'none';
+}
+
+/**
+ * function copyTextToClipboard() copies the text in the input field
+ * with the id 'session-id-field' into the clipboard.
+ */
+function copyTextToClipboard() {
+  const /** HTMLElement */ sessionIdElement =
+      document.getElementById('session-id-field');
+  sessionIdElement.select();
+  document.execCommand('copy');
+  alert('Copied the text: ' + sessionIdElement.value);
```

Comment:
This is more of a preference thing, but it's probably not necessary to alert here.